www/faq.txt

```diff
@@ -359,7 +359,7 @@ done
 
 for i in ${!tag_keys[@]} ; do
     key="${tag_keys[$i]}"
-    # they kebinding is as follows: the or executes the commands seperated by
+    # the keybinding is as follows: the or executes the commands separated by
```

Comment:
"is as follows" sounds wrong, too, not sure what is meant here

Reply:
I just want say that I'm about to explain the following `keybind` lines.

Reply:
maybe "works" instead of "is"?

Reply:
Yes!

Reply:
Done